package.json

```diff
@@ -266,7 +267,8 @@
     "node": ">= 0.8.x"
   },
   "scripts": {
-    "test": "make test-all"
+    "preinstall": "node scripts/fake-karma-mocha-peer-dep.js",
```

Comment:
`karma-mocha` has a peer dep on `mocha`, which is a problem.  we want it to use, well, _this_ mocha, so we just symlink `node_modules/mocha` to the root.  

this is causing a problem atm because we're not publishing the `scripts` dir.  I'll figure out a solution; it doesn't need to run anyway unless you're running `npm install` from a working copy